frontend/src/scenes/experiments/Experiment.tsx

```diff
@@ -388,7 +424,7 @@ export function Experiment(): JSX.Element {
                                             {' '}
                                             {variant}:
                                             <CodeSnippet language={Language.JavaScript}>
-                                                {`posthog.feature_flags.override({'${newExperimentData?.feature_flag_key}': '${variant}'})`}
+                                                {`posthog.feature_flags.override({'${experimentData.feature_flag_key}': '${variant}'})`}
```

Comment:
I renamed all of these to use `experimentData`, since `newExperimentData` isn't populated / doesn't make sense for running experiments.